### ISSUE TITLE
Slider: make model updates consistent for normal and discrete

### ISSUE THREAD

**Issue by ngraef** [angular/material#2060]
Demo: http://codepen.io/anon/pen/ZYPBaw

The normal slider continuously updates the model during a drag, while the discrete slider updates only after releasing the slider handle. They should have the same behavior, preferably updating on drag. Or perhaps this should be configurable like [`ngModelOptions`](https://docs.angularjs.org/api/ng/directive/ngModelOptions)?

**Comment by ngraef** [comment]
@ThomasBurleson Could you explain the reason for the "won't fix" decision?
